lib/screens_web/controllers/v2/screen_api_controller.ex

```diff
@@ -24,6 +24,7 @@ defmodule ScreensWeb.V2.ScreenApiController do
     screen_side = params["screen_side"]
     rotation_index = params["rotation_index"]
     triptych_pane = params["pane"]
+    triptych_player_name = params["player_name"]
```

Comment:
If we ever want to log DUP player names, we may want to consider renaming these params on the client. Just a thought, can't do that now.

Reply:
Oh hmm, we _could_... I could re-key the param over in `show_triptych/2` before passing off to `show/2`.

Reply:
Totally up to you. I think it will only be unclear once we add player name to params for DUPs. But I don't see us needing to do that in the near future.

Reply:
I'll leave it be for now.